Ginger/Ginger/Environments/EnvsListPage.xaml.cs

```diff
@@ -61,7 +63,10 @@ private void SetGridView()
             view.GridColsView = new ObservableList<GridColView>();
 
             view.GridColsView.Add(new GridColView() { Field = ProjEnvironment.Fields.Name, WidthWeight = 200 });
-            view.GridColsView.Add(new GridColView() { Field = ProjEnvironment.Fields.ReleaseVersion, WidthWeight = 80, Header = "Release Version" });
+
```

Comment:
this is the page in which all is readOnly?

Reply:
yes this is read only page